src/components/organisms/NotificationsManager/Notification.tsx

```diff
@@ -60,7 +62,7 @@ const Notification: React.FC<NotificationProps> = props => {
           <S.CopyOutlined onClick={onCopyToClipboard} />
         </Tooltip>
         <S.MessageBodyContainer>
-          <S.TitleSpan>{title}</S.TitleSpan>
+          <S.TitleSpan>{enhanceErrorMessage(title)}</S.TitleSpan>
           <S.MessageSpan>
             <NotificationMarkdown notification={notification} type={notificationType} />
           </S.MessageSpan>
```

Comment:
Same as above here in `Notification`, this should be changed `notification={alert} -> notification={{...alert, message: enhanceErrorMessage(alert.message)}}`